src/model/request-record.js

```diff
@@ -11,40 +11,48 @@ class RequestRecord {
 
   /**
    * Get other field from airtable.
-   * @param field Field to get from airtable record
-   * @returns {*}
+   *
+   * @param {object} field to get from airtable record
+   * @returns {*} The Airtable field.
    */
   get(field) {
     return this.airtableRequest.get(field);
   }
 
   /**
    * ID of the record
+   *
    * @type {string}
+   * @returns {number} The record ID.
```

Comment:
`@type` isn't sufficient for getters?

Reply:
Yeah good point. Updating to use `type` instead.